src/app/qgslabelinggui.cpp

```diff
@@ -229,7 +233,18 @@ void QgsLabelingGui::setLayer( QgsMapLayer *mapLayer )
   wrapCharacterEdit->setText( lyr.wrapChar );
   mAutoWrapLengthSpinBox->setValue( lyr.autoWrapLength );
   mAutoWrapTypeComboBox->setCurrentIndex( lyr.useMaxLineLengthForAutoWrap ? 0 : 1 );
-  mFontMultiLineAlignComboBox->setCurrentIndex( lyr.multilineAlign );
+
+  if ( ( int ) lyr.multilineAlign < mFontMultiLineAlignComboBox->count() )
+  {
+    mFontMultiLineAlignComboBox->setCurrentIndex( lyr.multilineAlign );
+  }
+  else
+  {
+    // the default pal layer settings for multiline aglinment is to follow label placement, which isn't always available
```

Comment:
Alignment

Reply:
Hmm wondering why it didn't trigger a spelling warning.